pkg/messaging/mqtt/pubsub.go

```diff
@@ -191,6 +206,21 @@ func newClient(address, id string, timeout time.Duration) (mqtt.Client, error) {
 	return client, nil
 }
 
+//? Prototype, removed the proto.Unmarshall()
```

Comment:
What does this mean?

Reply:
There was an error while unmarshalling the message into messaging.Message
I think it was because the test only contained the payload, and not the entire message